src/ElectricMoves.java

```diff
@@ -0,0 +1,33 @@
+public class ElectricMoves extends Moves {
+    public ElectricMoves(int moveAttack, int moveAccuracy, int moveDamage) {
+        super(moveAttack, moveAccuracy, moveDamage);
+    }
+
+    //    Electric
+
+    public void electroBall(){
+        setMoveAttack(15);
```

Comment:
Je zet nu elke keer opnieuw de waarde van deze klasse. Dat is misschien niet de beste methode. Het is misschien beter om in plaats van overerving, aggregatie toe te passen. Dus dat je niet van Moves extend, maar dat je een lisjt van Moves in deze klasse zeten en daat een `Move electroBall = new Move(15,15,15,"ffwooosshh")` in zet. En natuurlijk kun je ook de andere aanvallen er zo inzetten.